### PR TITLE
MM New MetaDB Derived Table Instance tags

### PR DESCRIPTION
**Pull request by jenmawe**
This derived table extracts tags from the instance record. Please note there are no tags in the instance records when this was tested on folio_juniper

### CODE REVIEW

**Comment by nassibnassar** [comment]
- [ ] Pull request has a complete description
- [ ] Query runs without errors
- [ ] Query output is correct
- [ ] Query logic is clear and well documented
- [ ] Query is readable and properly indented
- [ ] Table and column names are in all-lowercase
- [ ] Quotation marks are used only where necessary
- [ ] Query begins with user documentation in comment lines
- [ ] File name is listed in `runlist.txt` after dependencies
- [ ] All columns have indexes
- [ ] Table is vacuumed and analyzed

File: sql_metadb/derived_tables/runlist.txt [hunk 1]
```diff
@@ -26,6 +26,7 @@ instance_formats.sql
 instance_editions.sql
 instance_languages.sql
 instance_nature_content.sql
+instance_tags.sql
```
Comment (stdombek) [inline sql_metadb/derived_tables/runlist.txt:29]: There are no dependencies on other derived tables. So it should be sorted alphabetically. See also discussion in PR #638

**Comment by adoerrer** [comment]
* [x]  Pull request has a complete description
     * [x]  Query runs without errors
     * [ ]  Query output is correct
     * [x]  Query logic is clear and well documented
     * [x]  Query is readable and properly indented
     * [x]  Table and column names are in all-lowercase
     * [x]  Quotation marks are used only where necessary
     * [x]  Query begins with user documentation in comment lines
     * [x]  File name is listed in runlist.txt after dependencies
     * [x]  All columns have indexes
     * [x]  Table is vacuumed and analyzed
 
 
As @stdombek mentioned there is no test data to verify if the output is correct.

@jenmawe I did not find any option to add tags to an instance via UI. It is only possible for holdings and items. How can tags be added to an instance? If this is not possible this derived table might be obsolete.

**Review by adoerrer — COMMENTED** [review]
Looks good and without errors, but there's no test data to verify its output.

**Comment by jenmawe** [comment]
@adoerrer I thought that the ability to add tags to instances came with Lotus. But I think this might be Morning Glory for instances. The work is done and that's why we can see this in the API documentation. Tags has been implemented both in the back end and UI for holdings and items.

**Review by adoerrer — APPROVED** [review]
Thank you, for clarifying this @jenmawe. 
I would propose to approve this query since it does not run into errors.

**Review by stdombek — APPROVED** [review]
I agree with @adoerrer.

**Pull request merged**